src/core/algorithms/md/hymd/similarity_data.cpp

```diff
@@ -42,26 +58,43 @@ std::pair<SimilarityData, std::vector<bool>> SimilarityData::CreateFrom(
         }
         auto [lhs_ccv_id_info, indexes] = measure->MakeIndexes(
                 std::move(data_info_left), std::move(data_info_right), right_pli.GetClusters());
-        lhs_size_to_index.emplace_back(lhs_ccv_id_info.lhs_to_rhs_map.size(), column_match_index++);
-        column_matches_info.emplace_back(std::move(indexes), left_col_index, right_col_index);
-        all_lhs_ccv_ids_info.push_back(std::move(lhs_ccv_id_info));
-        short_sampling_enable.push_back(measure->IsSymmetricalAndEqIsMax());
+        lhs_size_to_index.emplace_back(lhs_ccv_id_info.lhs_to_rhs_map.size(),
+                                       lhs_ccv_id_info.rhs_to_lhs_map.size(), column_match_index++,
+                                       0);
```

Comment:
```suggestion
                                       lhs_ccv_id_info.rhs_to_lhs_map.size(), column_match_index++);
```
Just don't supply the argument if it's only set when it's used